### PR TITLE
[frontend] Gets accessToken every open of the app and added logout fu…

### PR DESCRIPTION
**Pull request by yan-alan**
Also stores refresh token, grabs a new access token at every launch of app!

### CODE REVIEW

File: frontend/screens/AlbumScreen.tsx [hunk 1]
```diff
@@ -230,7 +230,7 @@ const AlbumScreen = ({ route, navigation }) => {
       return (
         <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
           <Text style={{ fontSize: 18 }}>
-            Access token outdated!
+            {data.message}
```
Comment (yan-alan) [inline frontend/screens/AlbumScreen.tsx:233]: Not alwas an access token error, could be ML server can't be reached from Node, etc.

Comment (voctory) [reply frontend/screens/AlbumScreen.tsx:233]: 💀

**Review by voctory — APPROVED** [review]
LGTM

**Review by voctory — CHANGES_REQUESTED** [review]
Nevermind, found an edge case where after the user disconnects, they can exit and relaunch the app. Token is still associated with the app and not deleted; user is still 'logged in.'

**Comment by yan-alan** [comment]
Alright should've fixed that! Good catch

**Review by voctory — APPROVED** [review]
Fix is working! LGTM

**Pull request merged**